.travis.yml

```diff
@@ -28,6 +31,12 @@ matrix:
       env: SYMFONY_VERSION='3.0.*'
     - php: 7.0
       env: DEPENDENCIES='dev'
+    - php: hhvm
+      env: XDEBUG_DISABLE=''
+
+before_install:
+  # Remove xdebug as per https://getcomposer.org/doc/articles/troubleshooting.md#xdebug-impact-on-composer
+  - eval "$XDEBUG_DISABLE"
```

Comment:
IMO `phpenv config-rm xdebug.ini || true` will be more readable than all the stuff happening with environmental variables